src/components/listItems/userCourseListItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import { UserCourse, Course } from 'devu-shared-modules'
+
+import { prettyPrintDate } from 'utils/date.utils'
+
+import styles from './userCourseListItem.scss'
+
+import ColorHash from 'color-hash'
+
+type Props = {
+  userCourse: UserCourse
+  course: Course
+}
+
+const colorHash = (input: string) => {
```

Comment:
Ya know, my first thought was where you were getting the colors from here. This is classy as hell. Really cool idea. 

This is fine for now. But it's very likely that we'll want to pull this out into it's own little utils function so that we can have the same color being used for the course where ever we want rather than just here.

Again this is totally fine for now though. No need to unnecessarily abstract until we need it.

Reply:
I assigned myself issue #14 as well, so once we get a working design should I add it as a utils function for that pull request?